### PR TITLE
Delist 'Start Building RESTful Microservices using Akka HTTP with Scala'

### PR DESCRIPTION
**Pull request by psilospore**
I checked out this book after seeing it on the docs and found it to be poorly written. I don't think this book should be listed as a resource as it's not very helpful for readers of any level. I'm not sure if there are standards for what makes a resource good enough to be listed but if there were any I don't believe this book would meet the criteria.

### CODE REVIEW

**Comment by lightbend-cla-validator** [comment]
Hi @psilospore,

Thank you for your contribution! We really value the time you've taken to put this together.

Before we proceed with reviewing this pull request, please sign the Lightbend Contributors License Agreement:

[http://www.lightbend.com/contribute/cla](http://www.lightbend.com/contribute/cla)

**Review by ktoso — COMMENTED** [review]
Hi and thanks for bringing this up @psilospore.
I got the "book" and agree it is of not so great quality, rather than a book one could have called it a pretty long blog post really. It does not seem to be really wrong so I don't want to de-list it since this page is also to serve our community to get together and be happy about the work they've done.

What I would instead do here is the following (and please amend the PR if you feel comfortable with the suggestion, would love to merge it then):

- put the mentioned book at the bottom of the list.
- put `  * [Reactive Design Patterns](https://www.reactivedesignpatterns.com/), by Roland Kuhn with Jamie Allen and Brian Hanafee, Manning Publications Co., ISBN 9781617291807, Feb 2017 ` and `* [Akka in Action](http://www.lightbend.com/resources/e-book/akka-in-action), by Raymond Roestenburg and Rob Bakker, Manning Publications Co., ISBN: 9781617291012, September 2016` to the top of the list and make them be "Recommended reads: " the others being "Other reads about Akka and the Actor model:"

WDYT?

File: akka-docs/src/main/paradox/additional/books.md [hunk 1]
```diff
@@ -1,10 +1,12 @@
...
+### Recommended reads
  * [Reactive Design Patterns](https://www.reactivedesignpatterns.com/), by Roland Kuhn with Jamie Allen and Brian Hanafee, Manning Publications Co., ISBN 9781617291807, Feb 2017
+ * [Akka in Action](http://www.lightbend.com/resources/e-book/akka-in-action), by Raymond Roestenburg and Rob Bakker, Manning Publications Co., ISBN: 9781617291012, September 2016
+### Other reads about Akka and the Actor model
+ * [Akka Cookbook](https://www.packtpub.com/application-development/akka-cookbook), by Héctor Veiga Ortiz & Piyush Mishra, PACKT Publishing, ISBN: 9781785288180, May 2017
+
  * [Mastering Akka](https://www.packtpub.com/application-development/mastering-akka), by Christian Baxter, PACKT Publishing, ISBN: 9781786465023, October 2016
  * [Learning Akka](https://www.packtpub.com/application-development/learning-akka), by Jason Goodwin, PACKT Publishing, ISBN: 9781784393007, December 2015
  * [Akka in Action](http://www.lightbend.com/resources/e-book/akka-in-action), by Raymond Roestenburg and Rob Bakker, Manning Publications Co., ISBN: 9781617291012, September 2016
```
Comment (renatocaval) [inline akka-docs/src/main/paradox/additional/books.md]: This is appearing twice. In the recommended reads and here.

**Review by ktoso — APPROVED** [review]
LGTM thanks

**Comment by ktoso** [comment]
Removed the duplicate

**Pull request merged**